packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -70,12 +71,15 @@ export class BlobHandle implements IFluidHandle<ArrayBufferLike> {
 		public readonly path: string,
 		public readonly routeContext: IFluidHandleContext,
 		public get: () => Promise<any>,
+		onAttachGraph: () => void,
```

Comment:
```suggestion
		private readonly onAttachGraph: () => void,
```
then remove the definition, and the varibalbe setting below